src/Form/Form.js

```diff
@@ -1,38 +1,92 @@
 import './Form.css';
 import { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { mockPosters } from '../mockData'
 
-const Form = () => {
+const Form = ({setPoster}) => {
 
   const [formData, setFormData] = useState({
     //need to add an id in here somewhere
-    type: '',
-    imgLink: '',
     quote: '',
+    image: '',
+    type: '',
   })
 
-  const handleSubmit = () => {
-    //code here for what to do when form is submitted
+  const handleChange = (event) => {
+    const { name, value } = event.target
+  
+    setFormData(prevState => {
+      return {
+        ...prevState, 
+        [name]: value
+      }
+    })
+  }
+
+  const handleSubmit = (event) => {
+    setPoster(formData)
   }
 
   return (
-    <div id='form-page' >
+    <main id='form-page' >
       <h2>make your own quote!</h2>
-      <p>Your quote poster will be added to the quotes and can be generated randomly by future users.</p>
+      {/* <p>Your quote poster will be added to the quotes and can be generated randomly by future users.</p> */}
       <form onSubmit={handleSubmit}>
         <label htmlFor='type-of-poster'>1. select your quote type</label>
         <div className='buttons'>
-          <button className='form-button' id='wholesome-button'>🥹 wholesome quote</button>
-          <button className='form-button' id='chaotic-button'>😈 chaotic quote</button>
+          <label htmlFor="wholesome-radio" className='form-button' id='wholesome-button'>
```

Comment:
Why is it important to use radio buttons? I just don't love the styling and we can tie there values to the same state to get the back and forth effect - then add a little of our own styling to indicate which button is clicked?